b3sum/src/main.rs

```diff
@@ -118,6 +124,19 @@ fn write_output(mut output: blake3::OutputReader, mut len: u64) -> Result<()> {
     Ok(())
 }
 
+fn write_raw_output(mut output: blake3::OutputReader, mut len: u64) -> Result<()> {
+    let mut stdout = std::io::stdout();
```

Comment:
We should probably call `lock()` here, since we're about to use this in a loop.

Reply:
Done.